hack/make-rules/verify.sh

```diff
@@ -106,10 +107,13 @@ function is-quick {
 function is-explicitly-chosen {
   local name="${1#verify-}"
   name="${name%.*}"
+  index=0
   for e in ${WHAT}; do
     if [[ "${e}" == "${name}" ]]; then
+      TARGET_LIST[${index}]=""
       return
     fi
+    index=$(($index + 1))
```

Comment:
$/${} is unnecessary on arithmetic variables.

```suggestion
    index=$((index + 1))
```